src/App.tsx

```diff
@@ -4,13 +4,14 @@ import './App.scss';
 import todosFromServer from './api/todos';
 import usersFromServer from './api/users';
 
+import { TodoList } from './components/TodoList';
+
```

Comment:
remove unnecessary lines
```suggestion
import { TodoList } from './components/TodoList';
```